package.json

```diff
@@ -24,7 +24,9 @@
     "url": "https://github.com/SebastianUrbano/protractor-workshop-2020/issues"
   },
   "homepage": "https://github.com/SebastianUrbano/protractor-workshop-2020#readme",
-  "dependencies": {},
+  "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2"
```

Comment:
dev dependency

Reply:
It's done

Reply:
En qué archivo y parte se le podría indicar que instale las dev dependencies o dependencies? Supongo que tiene utilidad a la hora de indicar en otras maquinas (Ejemplo producción) qué necesita instalar realmente, es correcto?

Reply:
Es correcto, con npm install --only=prod` (o `--only=production`) se instalarán sólo las dependencias de producción